packages/drivers/local-driver/src/localDocumentStorageService.ts

```diff
@@ -82,13 +100,51 @@ export class LocalDocumentStorageService implements IDocumentStorageService {
 		summary: ISummaryTree,
 		context: ISummaryContext,
 	): Promise<string> {
+		// const createDocument = CreateDocumentInStorage.getInstance();
+		// if (createDocument) {
+		// 	await createDocument.createDocument(summary);
+		// }
+		if (this.delayedDocumentCreation) {
```

Comment:
I think you could call createDocument() based on context.referenceSequenceNumber === 0, since that should only be the case on the first summary.